### DISCUSSION TITLE
Any way to exclude localhost when using the Events API?

### DISCUSSION THREAD

**Discussion by markcellus** [plausible/analytics#1318] [Anything Else]
I've started testing tracking all of my events by making a call to my local server, which proxies to the new [Events API](https://plausible.io/docs/events-api), which works great! It's allowed me to eliminate all the plausible client-side scripts. However, I'd like to exclude all events coming from the localhost hostname. 

I can obviously do this on client-side by just sniffing the `window.location.hostname` value and, if it is `localhost`, don't call the events endpoint. But I wonder if there's a way to do this via the Events API? Or perhaps allow a way to exclude stats localhost hostname in the dashboard?

Thanks for any information!

**Comment by OriyomiOG** [comment]
Hey good day

**Comment by ukutaht** [comment]
@markcellus Sorry, this option doesn't exist at the moment. The way it's currently implemented is always on the client side.

**Reply by markcellus** [reply]
No problem. I can do the localhost check on the client for now. Thanks for the information. 👍
